src/components/ReportingTile.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="reporting-tile">
+    <p class="headline">
+      Years Reported
+      <span class="score">{{ reportedYearsCount }}/{{ reportingHistory.length }}</span>
+      <span class="grade">{{ grade }}</span>
+    </p>
+    <ul>
+      <li
+        v-for="item in reportingHistory"
+        :key="item.year"
+        class="reporting-tile-item"
+      >
+        <div
+          class="marker"
+          :class="{ '-reported': item.isReported }"
+        >
+          <img
+            v-if="item.isReported"
+            src="/checkmark.svg"
+            :alt="`${item.year} data reported`"
+            class="reported"
+          >
+          <img
+            v-else
+            src="/cross.svg"
+            :alt="`${item.year} data not reported`"
+          >
+        </div>
+        <p>{{ item.year }}</p>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+import { IHistoricData } from "../common-functions.vue";
+
+import { LatestDataYear } from "../constants/globals.vue";
+
+@Component
```

Comment:
Can you add a quick doc block, and note some good examples to test this with? We'll be writing some unit and E2E tests later, and that'll be helpful.